src/graph/test/GroupByLimitTest.cpp

```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/test/TestEnv.h"
+#include "graph/test/TestBase.h"
+#include "graph/test/TraverseTestBase.h"
+#include "meta/test/TestUtils.h"
+
+DECLARE_int32(load_data_interval_secs);
+
+namespace nebula {
+namespace graph {
+
+class GroupByLimitTest : public TraverseTestBase {
```

Comment:
BTW, why name this file as `GroupBy` `Limit` test?
I think it is not about `GroupBy`

Reply:
it use the same test file with #749, it will combine if it merge